scripts/generate-stylesheet.ts

```diff
@@ -0,0 +1,48 @@
+import { ColorRegistry } from '../packages/main/src/plugin/color-registry';
+import type { ApiSenderType } from '../packages/main/src/plugin/api';
+import type { ConfigurationRegistry } from '../packages/main/src/plugin/configuration-registry';
+import type { ColorInfo } from '../packages/api/src/color-info';
+import { writeFileSync } from 'node:fs';
+
+const apiSenderTypeMock: ApiSenderType = {
+  send: () => {},
+  receive: () => ({ dispose: () => {} }),
+};
+
+const configurationRegistryMock: ConfigurationRegistry = {
+  onDidChangeConfiguration: () => {},
+} as unknown as ConfigurationRegistry;
+
+// Create color registry
+const registry = new ColorRegistry(apiSenderTypeMock, configurationRegistryMock);
+registry.init();
```

Comment:
can we have another constructor in ColorRegistry ?

Reply:
`TS2392: Multiple constructor implementations are not allowed.` apparently no

Reply:
it's possible but you need to handle all of them with the same block

Reply:
anyway it's a low comment, as the goal is to move it to a separate file soon

Reply:
Ok